lib/kafka-worker.rb

```diff
@@ -101,7 +120,9 @@ def handle(message)
     end
 
     def on_error(message, err)
-      logger.error("#{self.class.name} failed on message: #{message.inspect} with error: #{err}")
+      error_message = "#{self.class.name} failed on message: #{message.inspect} with error: #{err}"
+      logger.error(error_message)
+      Rollbar.error(error_message)
```

Comment:
let's actually put all this code into the `rescue` block itself, so the job can overwrite `on_error` without necessarily having to call `super` to perform any cleanups or so